lib/fluent/plugin/in_tail.rb

```diff
@@ -593,13 +592,11 @@ def on_notify
           begin
             if @inode != inode || fsize < @fsize
               # rotated or truncated
+              io = FileWrapper.open(@path)
```

Comment:
@sonots : good catch - thanks!
@repeatedly : I'll update the pull request with your suggestion